compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/TypeChecker.java

```diff
@@ -8219,4 +8229,14 @@ private FieldInfo(List<BType> types, boolean required, boolean readonly) {
             this.readonly = readonly;
         }
     }
+
+    private static class TypeSymbolPair {
+        private BVarSymbol fieldSymbol;
+        private BType determinedType;
+
+        public TypeSymbolPair(BVarSymbol fieldSymbol, BType determinedType) {
```

Comment:
Since it is TypeSymbolPair, should the type be the first param? :smiley: Just wondering.